lisa/tools/date.py

```diff
@@ -25,3 +25,15 @@ def current(self, no_error_log: bool = True) -> datetime:
                 f"'Date' return non-zero exit code: {command_result.stderr}"
             )
         return parser().parse(command_result.stdout)
+
+    def set(self, time_str: str, format: str = "") -> datetime:
```

Comment:
Let it pass in the Python format datetime object. This code format it here, unless we need to test the date format ourselves.